app/Http/Controllers/api/Auth/SocialLoginController.php

```diff
@@ -28,15 +28,11 @@ public function callback($service)
     {
         try {
             $serviceUser = Socialite::driver($service)->stateless()->user();
+            $email = $serviceUser->getEmail();
```

Comment:
What was the reason for moving the email call here? I don't think the getEmail function would ever throw an exception.

Reply:
ah, ok...I can take it out of the try block. I thought it could fail / throw an exception but I havent dug into the socialite docs.

Reply:
@fritsvt done